src/JuMP.jl

```diff
@@ -32,8 +32,8 @@ export
     affToStr, quadToStr, conToStr, chgConstrRHS,
 
 # Macros and support functions
-    @addConstraint, @addConstraints, @defVar,
-    @defConstrRef, @setObjective, addToExpression, @defExpr,
+    @addConstraint, @defConstraint, @addConstraints, @defConstraints,
```

Comment:
We could implement ``@defConstraint`` so that it returns a collection of constraints (or JuMPContainers), and ``@addConstraint`` as a wrapper around this that just adds all the constraints to a model.